src/components/molecules/Visualizer/Plugin/types.ts

```diff
@@ -362,8 +362,11 @@ export type Camera = {
     options?: CameraOptions,
     offset?: number,
   ) => void;
+  readonly getPixelDistance: (sample?: PixelDistanceSamplePosition) => number | undefined;
```

Comment:
Is it really a `sample`? It is actually the y-location, or verticalPosition, right? Can you think of a better name for the param?

Reply:
Updated to `verticalPosition`. I think the full name might be `verticalSamplePosition` and the type name might be `PixelDistanceVerticalSamplePosition` but this is really too long and `PixelDistanceSamplePosition` seems more meaningful than `PixelDistanceVerticalPosition` so i did not change the type name.

Reply:
Is there any reason you are wanting to keep Sample in the type name? For me I can't see why its a sample. (kind of related to my other comment, so you can reply to both there)

Reply:
Please check the reply below.